.pre-commit-config.yaml

```diff
@@ -1,22 +1,49 @@
+# See https://pre-commit.com for more information
```

Comment:
Might want to set the autoupdate frequency to monthly or quarterly.

https://pre-commit.ci/#configuration-autoupdate_schedule

Reply:
thx, fixed